anndata/h5py/__init__.py

```diff
@@ -21,9 +21,4 @@
    Dataset
    SparseDataset
 """
-from .h5sparse import File, Group, SparseDataset, _load_h5_dataset_as_sparse
-from h5py import Dataset, special_dtype
-
-# Problem: the H5py intersphinx is broken, and only contains e.g. `Dataset` directly.
-# So we can’t possibly link to it using :class:`Dataset`, since that will always find our version.
-Dataset.__doc__ = """Equivalent to :class:`h5py.Dataset <h5py:Dataset>`."""
+from .h5sparse import SparseDataset, _load_h5_dataset_as_sparse
```

Comment:
The docs still mention this exporting Group, Dataset, … which you should fix.

Reply:
This file has moved to `anndata/core/sparsedataset/__init__.py` and those references are removed. The docs there could be expanded though.